lib/game_stat.rb

```diff
@@ -0,0 +1,21 @@
+class Game_Stat
+  attr_reader :game_id, :team_id, :home_or_away, :won, :settled_in, :head_coach, :goals, :shots, :hits, :pim, :power_play_opportunities, :power_play_goals, :face_off_win_percentage, :giveaways, :takeaways
+
+  def initialize(game_stat)
+    @game_id = game_stat[:game_id]
```

Comment:
This is a join table between team_id and game_id.

This means that this GameStat is going to have a reference to both a team and a game.

In Rails land we have a keyword for it called `:belongs_to`.

```
teams
id | name
1  | "Blackhawks"
2  | "RedWings"

games
id | played_date
1  | "12/27/2019"

game_stats
id | game_id | team_id | goals
1  | 1              | 1            | 3
2  | 1             | 2            | 2
```

Blackhawks got three goals to two.